frontend/pages/admin/IntegrationsPage/cards/MdmSettings/components/EndUserMigrationSection/EndUserMigrationSection.tsx

```diff
@@ -124,31 +125,35 @@ const EndUserMigrationSection = ({ router }: IEndUserMigrationSectionProps) => {
   if (!config?.mdm.apple_bm_enabled_and_configured) {
     return (
       <div className={baseClass}>
-        <h2>End user migration workflow</h2>
+        <SectionHeader title="End user migration workflow" />
         <EmptyTable
           className={`${baseClass}__abm-connect-message`}
           header="Migration workflow for macOS hosts"
           info="Connect to Apple Business Manager to get started."
-          primaryButton={<Button onClick={onClickConnect}>Connect</Button>}
+          primaryButton={
+            <Button variant="brand" onClick={onClickConnect}>
```

Comment:
Random extra fixes: Added variant="brand" to several buttons because they were missing on hover color change